shared/ui-components/src/page_components/DashBlocks.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react"
+import "./DashBlocks.scss"
+
+const DashBlocks = (props: { children: JSX.Element | JSX.Element[] }) => (
```

Comment:
Let's avoid the use of `JSX.Element` per https://github.com/bloom-housing/bloom/issues/450